libs/community/langchain_community/embeddings/baichuan.py

```diff
@@ -37,9 +37,16 @@ class BaichuanTextEmbeddings(BaseModel, Embeddings):
     """
 
     session: Any  #: :meta private:
-    model_name: str = "Baichuan-Text-Embedding"
-    baichuan_api_key: Optional[SecretStr] = None
+    model_name: str = Field(default="Baichuan-Text-Embedding", alias="model")
+    baichuan_api_key: Optional[SecretStr] = Field(default=None, alias="api_key")
     """Automatically inferred from env var `BAICHUAN_API_KEY` if not provided."""
+    chunk_size: int = 16
```

Comment:
How was this chosen / compare with limits on the underlying API? Is it possible it's too low (or high)?

Reply:
The batch size is equal to 16 because according to [Baichuan Embedings docs](https://platform.baichuan-ai.com/docs/text-Embedding), the document specifies that the maximum single input is the number of texts, which is 16.